### ISSUE TITLE
Long literal include have broken border

### ISSUE THREAD

**Issue by andreacassioli** [sphinx-doc/sphinx#2450]
I have some `literal-include` that span more than one page. In that case the border around the text looks broken: in the first page the left border is missing, whereas it is there on the other pages.

For example

![screenshot from 2016-04-14 11 29 43](https://cloud.githubusercontent.com/assets/6704112/14523531/7c720e4e-0234-11e6-8dcf-508fbc92e379.png)

As you see the second box has a left border. The corresponding Tex is

```
\def\SphinxLiteralBlockLabel{\label{license:fplib-license}}
\SphinxSetupCaptionForVerbatim{literal-block}{\titleref{fplib} license.}
\begin{Verbatim}[commandchars=\\\{\}]
/****************************************************************
 *
 * The author of this software is David M. Gay.
 *
 * Copyright (c) 1991, 2000, 2001 by Lucent Technologies.
 *
 * Permission to use, copy, modify, and distribute this software for any
 * purpose without fee is hereby granted, provided that this entire notice
 * is included in all copies of any software which is or includes a copy
 * or modification of this software and in all copies of the supporting
 * documentation for such software.
 *
 * THIS SOFTWARE IS BEING PROVIDED \PYGZdq{}AS IS\PYGZdq{}, WITHOUT ANY EXPRESS OR IMPLIED
 * WARRANTY.  IN PARTICULAR, NEITHER THE AUTHOR NOR LUCENT MAKES ANY
 * REPRESENTATION OR WARRANTY OF ANY KIND CONCERNING THE MERCHANTABILITY
 * OF THIS SOFTWARE OR ITS FITNESS FOR ANY PARTICULAR PURPOSE.
 *
 ***************************************************************/
\end{Verbatim}
\let\SphinxVerbatimTitle\empty
\let\SphinxLiteralBlockLabel\empty
```

Any ideas?

**Comment by jfbu** [comment]
The produced LaTeX looks ok. I can't reproduce the problem. I tried to reconstitute

``` rst
TEST
====

FIRST
-----

Hello

* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo
* foo

.. literalinclude:: bar.c
   :language: guess
   :caption: Code implementing problem
```

but this does not produce the problem although the LaTeX is identical. (for this the `bar.c` file has all lines starting with a space; else there is some additional highlighting on output)

A minimal example with `rst` sources would be helpful.

**Comment by andreacassioli** [comment]
Oh my bad! I am so ashamed....

It is a viewer issue: if I zoom more the line appears....

Thank for the effort guys.

**Comment by jfbu** [comment]
PDF viewers have artefacts with rules (filled rectangular areas) sometimes ... for example when two rules are contiguous one can see very thin white space in-between, but in truth the gap is non existent as revealed by zooming. Depends on anti-aliasing settings, sometimes.

**Comment by jfbu** [comment]
@andreacassioli sorry for not more seriously handle this back then: on my laptop I could not reproduce the problem. Should be fixed by PR #2638.

**Comment by jfbu** [comment]
Thanks for reporting ! finally fixed as part of upcoming 1.4.4 release. Closing now.
